src/Stratis.Bitcoin.Features.PoA/PoAMinerSettings.cs

```diff
@@ -8,6 +8,8 @@ public class PoAMinerSettings
         /// <summary>Allows mining in case node is in IBD and not connected to anyone.</summary>
         public bool BootstrappingMode { get; private set; }
 
+        public bool DevMode { get; private set; }
```

Comment:
please add comment on why we need dev mode and what it will be used for

Reply:
Done.